doc/source/user_guide/timeseries.rst

```diff
@@ -319,6 +319,11 @@ which can be specified. These are computed from the starting point specified by
    pd.to_datetime([1349720105100, 1349720105200, 1349720105300,
                    1349720105400, 1349720105500], unit='ms')
 
+.. note::
+
+   The strings used to specify a ``unit`` are not the same are those by ``format``.
```

Comment:
I am not really sure what format refers to here, can you clarify

Reply:
Sure! I'm talking about the "%Y-%M-%D" -type strings. Is the new commit clearer?